packages/react-components/src/icons/shared-research.tsx

```diff
@@ -3,12 +3,16 @@ import { FC } from 'react';
 
 interface LibraryIconProps {
   readonly color?: string;
+  readonly size?: number;
 }
 
-const LibraryIcon: FC<LibraryIconProps> = ({ color = '#4D646B' }) => (
+const LibraryIcon: FC<LibraryIconProps> = ({
+  color = '#4D646B',
+  size = 24,
+}) => (
   <svg
-    width="24px"
-    height="24px"
+    width={size}
```

Comment:
is `24` and `24px` the same? just curious

Reply:
yes